mindmeld/auto_annotator.py

```diff
@@ -0,0 +1,661 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC, abstractmethod
+import re
+import logging
+import os
+import importlib
+from tqdm import tqdm
+import spacy
```

Comment:
we should add spacy to the requirements

Reply:
So far I have added "spacy>=2.3.0" to the requirements array in setup.py (Line 37). Which other files would I need to update as well?

Reply:
Ah, my bad 👍